presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java

```diff
@@ -38,7 +38,7 @@
     private static final String SYSTEM_MEMORY_GB = "system-memory-gb";
     private static final String TASK_MAX_DRIVERS_PER_TASK = "task.max-drivers-per-task";
     private static final String SHUFFLE_NAME = "shuffle.name";
-
+    private static final String HTTP_SERVER_ACCESS_LOGS = "http-server.enable-access-log";
```

Comment:
Let's start documenting these properties.

Reply:
Added comments to some of these.